doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst

```diff
@@ -155,15 +155,16 @@ To read the dictionary-based STM log output, do the following:
    * ``<app_name>`` is the application name.
    * ``<port>`` is the serial port used for output.
      Use ``nrfutil device list`` to list which serial ports are exposed by the development kit.
+   * The output can be either the console (``--stdout ascii``) or a file (the :file:`out.txt` file if ``--output-ascii out.txt``).
```

Comment:
This command is confusing. "ascii" is a format, not an output target. I would expect `--out stdout` or `--out path_to_file`. But I guess it is as it is.

Reply:
The help states:
```OUTPUT ARGUMENTS:
      --output-raw <PATH>
          Output the trace data to raw (binary) file format

      --output-ascii <PATH>
          Output the trace data to ascii file format (plain text)

      --stdout <OUTPUT_TYPES>
          Output the trace data to standard out.

          Select between one or more output types, separating each value with comma.

          Possible values:

          - binary:  Display binary data

          - network: Display network data

          - tls:     Display TLS data

          - ascii:   Display ASCII data

          Possible values:
          - binary:  Display binary data
          - network: Display network data
          - ascii:   Display ASCII data
          - tls:     Display TLS data
          -
```

Reply:
Yeah, these commands are confusing. As a user, I would expect:

`--format ascii|binary|network|tls` printing data to stdout. Then I could simply do `> path_to_file` to redirect the output stream. If console redirection is not enough, a common approach is `--output stdout|path_to_file` that can be used to redirect the stream from within the program.

Reply:
Reported in https://nordicsemi.atlassian.net/browse/NRFU-1251